cmd/syncRules.go

```diff
@@ -116,6 +123,45 @@ func loadRego(path string) (*regoFile, error) {
 	return &rego, err
 }
 
+var familyToUUIDCache map[string]string = nil
+
+func familyToUUID(family string) (string, bool) {
+	if familyToUUIDCache == nil {
+		client, auth := getClient()
+		familyToUUIDCache = make(map[string]string)
+
+		params := families.NewListFamiliesParams()
+		var currentOffset int64 = 0
+		var maxItems int64 = 100
+		params.MaxItems = &maxItems
+		params.Offset = &currentOffset
+
+		for {
+			if famList, err := client.Families.ListFamilies(params, auth); err == nil {
+				for _, famInfo := range famList.Payload.Items {
+					if !(famInfo.Source == "CUSTOM" || famInfo.ID == "Custom") {
+						continue
+					}
+					familyToUUIDCache[famInfo.Name] = famInfo.ID
+				}
+
+				currentOffset += maxItems
+				if currentOffset >= famList.Payload.Count {
+					break
```

Comment:
`Count` is the number of returned results so I think this can be incorrect.  You can copy some loop boilerplate from [here](https://github.com/fugue/regula/blob/5559b0ffe263ea984e4f94ab1b9a9b486992f9be/cmd/scan.go#L139)

Reply:
That was my initial assumption as well, but it turns out `Count` is the *total* number of results, not the number returned in this result set; change `maxItems` to 10 to test (or whatever; there are 16 default families, or at least there were on my account).

This could be different from other parts of the API (i.e., maybe there is a bug in the server generated count here?).  If you want I can go ahead and modify this to use the boilerplate you linked to; if nothing else I guess that would make the code more idiomatic.

Reply:
Oh, in that case my assumption may be wrong!  In either case, I think having idiomatic would be neat, and I guess making less assumptions about the returned values is maybe also a small advantage.